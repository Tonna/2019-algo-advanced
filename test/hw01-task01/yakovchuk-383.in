25
1 1 1 3 1 5 2 4 3 1 3 4 8 12 12 3 1 9 6 19 9 18 14 11
7
19 10
11 8
3 1
19 10
16 3
7 4
9 6
