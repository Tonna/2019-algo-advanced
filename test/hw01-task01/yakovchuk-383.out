28
22
3
28
18
12
11
